.github/workflows/dapr.yml

```diff
@@ -78,8 +78,17 @@ jobs:
     env:
       ARTIFACT_DIR: ./release
       DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+      HELMVER: v3.0.3
     runs-on: ubuntu-latest
     steps:
+      - name: Set up Helm ${{ env.HELMVER }}
+        uses: azure/setup-helm@v1
+        with:
+          version: ${{ env.HELMVER }}
+      - name: Generate helm chart manifest
+        run: make manifest-gen
+      - name: Rename helm chart manifest
+        run: mv ./dist/install/dapr.yaml $ARTIFACT_DIR/dapr-operator.yaml
```

Comment:
You are generating before clone repo :) publish build will be done in new vm where code doesn't exist.

move helm chart generation and copy helm chart manifest steps to line 105 (before download artifact)

Reply:
Oh that's right, moved those steps now.